src/mixer/mixer_functions.cpp

```diff
@@ -136,7 +136,7 @@ FunctionProperties<Periodic_function<double>> periodic_function_property_modifie
 {
     auto global_size_func = [](Periodic_function<double> const& x) -> double
     {
-        return x.ctx().unit_cell().omega();
+        return 1.0 / x.ctx().unit_cell().omega();
```

Comment:
and this is where you suggested to take `1.0`, instead of `1.0 / omega`, but I left it as one over omega because it corresponds with what QE does. We can make it 1.0, but then there should be an omega^2 term in `set new tolerance of iterative solver` to ensure QE and SIRIUS still set the same threshold for Davidson.

Reply:
I checked: you are  correct. 
`V_{Ha}({\bf G}) = 4\pi \frac{\rho({\bf G})} {{G^2}}` and so inner product has `\Omega` prefactor